index1.html

```diff
@@ -0,0 +1,104 @@
+                             <li class="content-liCc"><a href="#">CC compaign</a></li>
+                      </ul>
+               </nav>
+        </div>
+              </div>
+          </div>
+ </header>
+
+ <div class="welcom">
+  <h1 class="welcom-h1">Developed by young developers for the improvement of national education</h1>
+
+  <p class="welcom-p1">
+      
+  <p class="welcom-p2">
+   The integration of Artificial Intelligence (AI) in education assessment is a cutting-edge technological advancement that is revolutionizing the traditional way of correcting exams. With the use of AI, the process of grading exams becomes faster, more accurate, and less subjective. This technology uses algorithms that are programmed to recognize patterns and identify correct answers based on a set of criteria.
+  </p>
+  <p class="welcom-p3">
+     <span>Please contact us for more information and investment  <a href="#">zoumaniguimass12@gmail.com</a></span>
+   
+  </p>
+
+ </div>
+
+<div class="welcom1">
+
+   <div class="welcom11" >Topic goal</div>
+   <div class="welcom12" ></div>
+   <div class="welcom13" >Overall, furfur represents an exciting and rapidly-evolving field within AI in education, with the potential to revolutionize the way that students learn and teachers teach. As the technology continues to advance and improve, it is likely that we will see furfur and other AI-based systems play an increasingly important role in education at all levels.
+   </div>
+   <div class="welcom14" ><img class="welcom14img" src="images/fontAI1.jpg"  width="100%" height="100%" alt=""> </div>
+</div>
+<div class="dashlong"><hr></div>
+
+<div class="welcom2">
+
+ <div class="welcom21" >Meeting</div>
+ <div class="welcom22" ></div>
+ <div class="welcom23" >Furfur is a term commonly used in the context of AI in education to describe a computer-based learning system that is designed to personalize the learning experience of individual students. Furfur utilizes machine learning algorithms to analyze data on student performance, preferences, and interests, and then uses this information to generate customized learning paths for each student.
+ </div>
+ <div class="flexwelcom24">
+ <div class="welcom24" ><img src="images/r.png" width="100%" height="100%" alt=""></div>
+ <div class="welcom24" ><img src="images/fontAI2.jpg"  width="100%" height="100%" alt=""></div>
+</div>
+</div>
+
+
+
+  <div class="footer2">
+   <div class="footer21"> Revolutionizing Education Assessment: The Integration of Artificial Intelligence for Exam Correction</div>
+   <div class="footer22">  In addition to personalized learning paths, furfur may also incorporate other AI-based features such as adaptive assessment, virtual tutoring, and natural language processing. These features can help to further enhance the learning experience by providing real-time feedback, answering questions, and facilitating communication between students and teachers.</div>
+</div>
+</div>
```

Comment:
- [ ] Kindly use your logo in this footer, use the correct text color and the content should be centered to match the design guideline.

- Given Design:
- ![image](https://user-images.githubusercontent.com/60083437/227040253-194dec49-f728-4925-acf6-c8deedc1f191.png)